client/src/pages/Start.js

```diff
@@ -49,17 +52,16 @@ const Start = () => {
     // Execute the query on component load
     const { loading, data } = useQuery(GET_HUNTS)
 
-    // Use optional chaining to check if data exists and if it has a thoughts property. If not, return an empty array to use.
+    // Assigns the query data to "hunts" if available. If it is not available then it assigns an empty array. 
     const hunts = data?.hunts || [];
 
-    console.log(hunts)
-
+    // making sure the data is loaded before things start being assigned to the page. 
     if (loading) {
         return <h2 style={{ justifyContent:'center'}}>LOADING.....</h2>
     }
 
+    // Removes duplicates in the cities
     const cities = [...new Map(hunts.map(hunt => [hunt.city, hunt])).values()]
-    console.log(cities)
 
     return (
         <><motion.div
```

Comment:
Can we remove the fragment here? (Remove the `<>`?)

Reply:
I can look but likely no. Sometimes you need the fragment or the whole code block breaks!